manifests/params.pp

```diff
@@ -35,6 +35,12 @@
       $servicename      = 'dhcpd'
       $package_provider = undef
     }
+    'Archlinux': {
+      $dhcp_dir         = '/etc'
```

Comment:
So my prefered way in general is to stay with upstream. But in that case that would be very itchy. I would like to create a /etc/dhcp/ directory and throwevery file in it and simply adjust the default service file.